PyExpUtils/utils/cmdline.py

```diff
@@ -0,0 +1,11 @@
+from typing import Any, Optional, Sequence, Tuple
+
+def flagString(pairs: Sequence[Tuple[str, Optional[Any]]], joiner: str = '='):
+    s = ''
+    for i, pair in enumerate(pairs):
+        key, value = pair
+        if value is not None:
+            if i > 0: s += ' '
```

Comment:
Hmph, there's a spacing bug here in this case:
```
pairs = [
  ('--thing', None),
  ('--other', 'a'),
]
```
because `i` will be greater than 0 for the first non-None element.

Not mission critical so will ignore and add to my todo-list for now.